src/bundle/Resources/public/scss/_tables.scss

```diff
@@ -249,8 +240,8 @@
 }
 
 .ez-scrollable-table-wrapper {
-    overflow-x: auto;
```

Comment:
Little nitpicking - if you haven't changed here anything (and you didn't, as it's just moved line below), it shouldn't to avoid git change.